### PR TITLE
Charlie update

### PR DESCRIPTION
**Pull request by charlie-becker**
Changes to CAPE/Shear plot that is much more efficient and can plot many more storms much quicker. Additional aesthetic changes.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -5,11 +5,13 @@
                 "scipy",
                 "xarray",
                 "pandas",
+                "dask[dataframe]",
```
Comment (djgagne) [inline setup.py:8]: Good use of specifying a particular part of a package as as dependency. I didn't realize this was a thing until seeing it here.

File: hwtmode/interpretation.py [hunk 2]
```diff
@@ -136,48 +137,44 @@ def plot_top_activations(out_path, model_name, x_data, meta_df, neuron_activatio
...
+    plt.figure(figsize=(20,16))
     sns.set(font_scale=2)
-    sns.kdeplot(data=df, x='CAPE', y='6km Shear', hue='Neuron', fill=True, alpha=0.8, thresh=0.5, clip=(0, 8000))
-    plt.title(f'Storm Activations for Top {num_storms} Storms')
-    plt.savefig(f'{output_path}{model_name}/CAPE_Shear_{mode}.png', bbox_inches='tight')
-
+    sns.scatterplot(data=df, x='CAPE', y='6km Shear', hue='Neuron', alpha=0.5, size='Activation', sizes=(1, 200), edgecolors='k', linewidth=1)
+    sns.kdeplot(data=df, x='CAPE', y='6km Shear', hue='Neuron', fill=False, alpha=1, thresh=0.4, levels=3, clip=(0,6000), linewidths=8, legend=False)
+    plt.title(f'Storm Activations for Top {num_storms} Storms ({mode})')
+    plt.savefig(f'{out_path}/CAPE_Shear_{model_name}.png', bbox_inches='tight')
```
Comment (djgagne) [inline hwtmode/interpretation.py:174]: For file paths, I encourage you to use `join(out_path, f'CAPE_Shear_{model_name}.png')` from `os.path`. It automatically handles paths that do or do not provide the / at the end.

File: hwtmode/interpretation.py [hunk 3]
```diff
@@ -136,48 +137,44 @@ def plot_top_activations(out_path, model_name, x_data, meta_df, neuron_activatio
         plt.close()
     return
 
-def cape_shear_modes(neuron_activations, output_path, data_path, model_name, mode, num_storms=50):
+def cape_shear_modes(neuron_activations, output_path, data_path, model_name, mode, num_storms=5000):
     """
     Match specified number of top storms of each neuron, fetch storm patch,
```
Comment (djgagne) [inline hwtmode/interpretation.py:142]: Thank you for including the docstring with all relevant variables.

**Review by djgagne — APPROVED** [review]
I have a minor path formatting change request but otherwise everything looks good to merge.

File: hwtmode/interpretation.py (hunk 2 above)
Comment (charlie-becker) [reply hwtmode/interpretation.py]: Changed. Merging now.

**Pull request merged**